committer-tools/refresh_collaborators.py

```diff
@@ -79,7 +79,7 @@ def get_top_contributors(repo: Repository, committers: List[str]) -> List[str]:
     Get top contributors for the given repository excluding committers.
     """
     logging.info(f"Fetching contributors from the repository {REPO_KAFKA}")
-    one_year_ago: datetime = datetime.now() - timedelta(days=365)
+    one_year_ago: datetime = datetime.now() - timedelta(days=90)
```

Comment:
nit:

change `one_year_ago` to `three_months_ago`

Reply:
If the number of days here may change over time, using a more general name like `lookback_period` would be better.

Reply:
I believe this value won't change frequently, and it might even be the last time :) 
Therefore, modifying the variable seems more appropriate.